app/views/admin/action_pages/_tab_buttons.html.erb

```diff
@@ -1,50 +1,50 @@
 <!-- tab_buttons.html.erb -->
-      <ul class="nav nav-tabs responsive" id="tabs">
-        <li>
-          <%= link_to admin_action_pages_path do %>
-            <i class="icon ion-ios-arrow-back"></i> Back
-          <% end %>
-        </li>
-        <li class="active">
-          <a data-tab='contents' href="#contents" data-toggle="tab">
-            <i class="icon ion-document-text"></i> Content
-          </a>
-        </li>
-        <li>
-          <a data-tab='action' href="#action" data-toggle="tab">
-            <i class="icon ion-flash"></i> Action Settings
-          </a>
-        </li>
-        <li>
-          <a data-tab='photo' href="#photo-settings" data-toggle="tab">
-            <i class="icon ion-camera"></i> Photos
-          </a>
-        </li>
-        <li>
-          <a data-tab='social' href="#social-settings" data-toggle="tab">
-            <i class="icon ion-social-twitter"></i> Social
-          </a>
-        </li>
-        <li>
-          <a data-tab='email' href="#email-settings" data-toggle="tab">
-            <i class="icon ion-email"></i> Email
-          </a>
-        </li>
-        <li>
-          <a data-tab='victory' href="#victory" data-toggle="tab">
-            <i class="icon ion-flag"></i> Victory/Archive
-          </a>
-        </li>
-        <% if @actionPage.id %>
-        <li>
-          <a data-tab='analytics' href="#analytics" data-toggle="tab">
-            <i class="icon ion-arrow-graph-up-right"></i> Analytics
-          </a>
-        </li>
-        <% end %>
-        <li class="pull-right open-gallery">
-          <a href="#" data-toggle="modal" data-target="#myModal">
-            <i class="icon ion-images"></i> Open Gallery
-          </a>
-        </li>
-      </ul>
+<ul class="nav nav-tabs responsive" id="tabs">
+<li>
+  <%= link_to admin_action_pages_path do %>
+    <i class="icon ion-ios-arrow-back"></i> Back
+  <% end %>
+</li>
+<li class="active">
+  <a data-tab='contents' href="#contents" data-toggle="tab">
+    <i class="icon ion-document-text"></i> Content
+  </a>
+</li>
+<li>
+  <a data-tab='action' href="#action" data-toggle="tab">
+    <i class="icon ion-flash"></i> Action Settings
+  </a>
+</li>
+<li>
+  <a data-tab='photo' href="#photo-settings" data-toggle="tab">
+    <i class="icon ion-camera"></i> Photos
+  </a>
+</li>
+<li>
+  <a data-tab='social' href="#social-settings" data-toggle="tab">
+    <i class="icon ion-social-twitter"></i> Social
+  </a>
+</li>
+<li>
+  <a data-tab='email' href="#email-settings" data-toggle="tab">
+    <i class="icon ion-email"></i> Email
+  </a>
+</li>
+<li>
+  <a data-tab='victory' href="#victory" data-toggle="tab">
+    <i class="icon ion-flag"></i> Victory/Archive
+  </a>
+</li>
+<% if @actionPage.id %>
+<li>
+  <a data-tab='analytics' href="#analytics" data-toggle="tab">
+    <i class="icon ion-arrow-graph-up-right"></i> Analytics
+  </a>
+</li>
+<% end %>
```

Comment:
I feel like conditionals like this should have the content 'under' it indented as well.